mail_tracking/models/mail_thread.py

```diff
@@ -30,43 +30,58 @@ def _get_failed_message_domain(self):
     def message_post(self, *args, **kwargs):
         """Adds CC recipient to the message.
 
-        Because Odoo implementation avoid store cc recipients we ensure that
-        this information its written into the mail.message record.
+        Because Odoo implementation avoid store 'cc/to' recipients we ensure
+        that this information its written into the mail.message record.
         """
         new_message = super().message_post(*args, **kwargs)
         email_cc = kwargs.get('cc')
+        to_write = {}
         if email_cc:
-            new_message.sudo().write({
-                'email_cc': email_cc,
-            })
+            to_write['email_cc'] = email_cc
+        # This is here because Odoo only store recipients that have a partner.
+        to_write['email_to'] = kwargs.get('to')
+        new_message.sudo().write(to_write)
         return new_message
 
     @api.multi
     def message_get_suggested_recipients(self):
-        """Adds email Cc recipients as suggested recipients.
+        """Adds email 'extra' recipients as suggested recipients.
 
         If the recipient has a res.partner, use it.
         """
         res = super().message_get_suggested_recipients()
+        db_aliases = set(map(
+            lambda x: x['alias_name'],
+            self.env['mail.alias'].search_read([
+                ('alias_name', '!=', False),
+            ], ['alias_name'])))
+        self._add_extra_recipients_suggestions(res, 'email_cc', _('Cc'),
+                                               db_aliases)
+        self._add_extra_recipients_suggestions(res, 'email_to', _('Anon. To'),
+                                               db_aliases)
+        return res
+
+    def _add_extra_recipients_suggestions(self, suggestions, field_mail,
+                                          reason, alias_set=None):
         ResPartnerObj = self.env['res.partner']
-        email_cc_formated_list = []
+        email_extra_formated_list = []
         for record in self:
-            emails_cc = record.message_ids.mapped('email_cc')
-            for email in emails_cc:
-                email_cc_formated_list.extend(email_split_and_format(email))
-        email_cc_formated_list = set(email_cc_formated_list)
-        for cc in email_cc_formated_list:
-            email_parts = getaddresses([cc])[0]
+            emails_extra = record.message_ids.mapped(field_mail)
+            for email in emails_extra:
+                email_extra_formated_list.extend(email_split_and_format(email))
+        email_extra_formated_list = set(email_extra_formated_list)
+        for mail in email_extra_formated_list:
+            email_parts = getaddresses([mail])[0]
```

Comment:
Have you considered this?

```suggestion
        for local, domain in getaddresses(email_extra_formated_list):
```